safety/cli.py

```
@@ -52,6 +52,11 @@
 
 LOG = logging.getLogger(__name__)
 
+DEPRECATION_DATE = date(2024, 11, 1)
```

Ruff

Undefined name `date`

(F821)